PublicMapping/districtbuilder
# Add majority race metric to project sidebar

## Overview

- Adds the majority race metric to project sidebar
- Allows the user to pin majority race metric

### Checklist


### Demo

![image](https://user-images.githubusercontent.com/66973361/126186732-4da6995a-12d3-45ba-871c-d5fddfe3d726.png)



## Testing Instructions

- Open a project and toggle the expanded metrics
- Expect majority race is displayed at the end of demographic headers
- Pin the majority race metric
- Expect majority race to be displayed when expanded metrics are collapsed

Closes #830
